### DISCUSSION TITLE
Dịch teacher forcing

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#174] [Terminology]
Trong notebook  của chương 16 có một kỹ thuật gọi là `teacher forcing` để train RNN. Theo mình hiểu thì kỹ thuật này như sau: trong quá trình training ở mỗi timestep, không cần biết mô hình output ra giá trị gì thì input của timestep tiếp theo cũng lấy giá trị groundtruth.
![](https://miro.medium.com/max/640/1*U3d8D_GnfW13Y3nDgvwJSw.webp)

Mình nên dịch teacher forcing như thế nào?
@mlbvn/handson-ml

**Comment by cuongvng** [comment]
`nhồi sọ`? Có từ nào nhẹ nhàng hơn không nhỉ?

**Comment by rootonchair** [comment]
`học theo thầy` à?

**Comment by cuongvng** [comment]
`ép học`
